test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -157,37 +184,78 @@ create_configs_for_newer_release_branches()
 #############################################################
 run_format()
 {
-        branch_name=$1
-        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
-        echo "Running format in branch: \"$branch_name\""
-        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
-
-        cd "$branch_name/test/format"
-        flags="-1q $(bflag $branch_name)"
-
-        config_file=""
-
-        # Compatibility test for newer releases will have CONFIG file for each release
-        # branches for the upgrade/downgrade testing.
-        #
-        # Compatibility test for older and standalone releases will have the default config.
-        if [ "$newer" = true ]; then
-            config_file="-c CONFIG_${branch_name}"
+    branch_name=$1
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    echo "Running format in branch: \"$branch_name\""
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+    cd "$branch_name/test/format"
+    flags="-1q $(bflag $branch_name)"
+
+    config_file=""
+
+    # Compatibility test for newer releases will have CONFIG file for each release
+    # branches for the upgrade/downgrade testing.
+    #
+    # Compatibility test for older and standalone releases will have the default config.
+    if [ "${wt_standalone}" = true ] || [ $older = true ]; then
+        config_file="-c CONFIG_default"
+    else
+        config_file="-c CONFIG_${branch_name}"
+    fi
+
+    for am in $2; do
+        dir="RUNDIR.$am"
+        echo "./t running $am access method..."
+        ./t $flags ${config_file} -h $dir "file_type=$am"
+
+        # Remove the version string from the base configuration file. (MongoDB does not create
+        # a base configuration file, but format does, so we need to remove its version string
+        # to allow backward compatibility testing.)
+        (echo '/^version=/d'
+            echo w) | ed -s $dir/WiredTiger.basecfg > /dev/null
+    done
+    cd -
+}
+
+#############################################################
+# run_test_checkpoint:
+#       arg1: branch name
+#       arg2: access methods list
+#############################################################
+run_test_checkpoint()
+{
+    branch_name=$1
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    echo "Running test checkpoint in branch: \"$branch_name\""
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+    cd "$branch_name/test/checkpoint"
+    flags="-W 3 -D -p -x -n 100000 -k 100000 -C cache_size=100MB"
```

Comment:
Can a comment be added here to mention this set of flags is expected to corrupt the database in some way?

Reply:
Done.